src/components/v2/devtronStackManager/DevtronStackManager.component.tsx

```diff
@@ -24,8 +24,10 @@ import { ReactComponent as File } from '../../../assets/icons/ic-file-text.svg'
 import { ReactComponent as Chat } from '../../../assets/icons/ic-chat-circle-dots.svg'
```

Comment:
Can remove `ReactComponent as File` line number 23 not using anywhere

Reply:
Done